docs/resources/compute_keypair.md

```diff
@@ -12,6 +12,7 @@ Manages a keypair resource within HuaweiCloud. This is an alternative to `huawei
 resource "huaweicloud_compute_keypair" "test-keypair" {
   name       = "my-keypair"
   public_key = "ssh-rsa AAAAB3NzaC1yc2EAAAlJq5Pu+eizhou7nFFDxXofr2ySF8k/yuA9OnJdVF9Fbf85Z59CWNZBvcAT... root@terra-dev"
+  key_file   = "private_key.pem"
```

Comment:
The information about the private key is contained in the API response for creating an SSH key.
The information about the private key is not contained in the API response for importing an SSH key.

Reply:
Now, we support two examples to show how to import an SSH key or create an SSH key.